.yamato/wrench/pvp-exemptions.json

```diff
@@ -0,0 +1,24 @@
+{
+  "per_package": {
+    "com.unity.formats.alembic@2": {
+      "exempts": {
+        "PVP-25-1": {
+          "errors": [
+            "Runtime/Plugins/ARM64/abci.dll",
+            "Runtime/Plugins/x86_64/abci.dll"
+          ]
+        },
+        "PVP-26-4": {
+          "errors": [
+            "Runtime/Plugins/x86_64/abci.bundle/Contents/Info.plist.meta: Meta file without corresponding asset"
```

Comment:
Still needed ?

Reply:
Right, I need to remove it. Thanks for catching it.